app/cosi/README.md

```diff
@@ -0,0 +1,338 @@
+
+CoSi will by default search for a file "group.toml" in the default configuration folders
+which are `$HOME/.config/cosi/` for Linux systems and `$HOME/Library/cosi/` for
+mac systems. If CoSi did not find anything, the default mechanism is to search in the current
+directory.
+
+Once you have a valid group definition, you can sign a file using the `cosi sign’
+command.  Here is an example that uses our default public CoSi server group to
+sign the README.md file at the top of the cosi source directory:
+
+```bash
+cosi sign -g dedis_group.toml README.md
+```
+
+When collective signing completes,
+the resulting signature will be written to standard output by default.
+To write the signature written to a file,
+you may redirect output or use the the `-o` option:
+
+```base
+cosi sign -g dedis_group.toml -o README.sig README.md
+```
+
+To verify a collective signature, use the `cosi verify` command:
+  
+```bash
+cosi verify -g dedis_group.toml -s README.sig README.md
+```
+
+Verification can also take the signature from standard input:
+
+```bash
+cat README.sig | cosi verify -g dedis_group.toml README.md
+```
+
+In the current prototype, CoSi witness servers do not validate or check the 
+messages you propose in any way; they merely serve to provide transparency
+by publicly attesting the fact that they have observed and cosigned the message.
+A future CoSi release will add support for message validation plugins,
+by which the servers can apply application-specific checks to messages
+before signing off on them,
+e.g., to validate a [collectively signed blockchain](http://arxiv.org/abs/1602.06997).
+
+## Running Your Own CoSi Witness Server
+
+First you need to create a configuration file for the server including a 
+public/private key pair.
+You can create a default server configuration with a fresh 
+public/private key pair as follows:
+
+```bash
+cosi server setup
+```
+
+Follow the instructions on the screen. At the end, you should have two files:
+* One local server configuration file which is used by your cothority server,
+* One group definition file that you will share with other cothority members and
+  clients that wants to contact you.
+
+To run the server, simply type:
+```bash
+cosi server
+```
+
+### Custom Configuration Path
+
+The server will try to read the default configuration file; if you have put the
+file in a custom location, provide the path using:
+```bash
+cosi server -config path/file.toml
+```
+
+### Debugging Output
+
+You can also ask the server to print out some debugging messages by indicating
+a level. Using level 1 shows when a message gets signed:
+
+```bash
+cosi -d 1 server
+```
+
+## Creating a Collective Signing Group
+
+If you run several CoSi servers,
+you can concatenate their individual `group.toml` outputs
+to define your own cosigning group.
+You may optionally use any or all of our experimental
+[default CoSi servers](https://github.com/dedis/cosi/blob/master/dedis_group.toml)
```

Comment:
So we should remove them  (the links) for the time being then, since it does not exists yet.